### ISSUE TITLE
Setting 'angle' with a drawRect()

### ISSUE THREAD

**Issue by AndrewMcMahan** [caleb531/jcanvas#239]
I'm currently working on updating a project from using jCanvas 5.1 to the current version.   For the most part, it's not TOO bad, just a lot of things that used to be 'default' now have to be set explicitly.

However, there is one thing that I cannot figure out.    With the old version of jCanvas, the code I'm using would use ```drawRect()``` and actually set an 'angle' parameter, like so:
```
		$('canvas').drawRect({
			fillStyle: '#000',
			strokeStyle: '#000',
			x: x, y: y,
			width: 40 * seatScale, height: 40 * seatScale,
			angle: -1 * t
		});
```
Is there a way to duplicate the 'angle' parameter with the latest version of jCanvas?   As far as I can tell, setting this parameter has no effect in the latest version.

---
#### Notes
1.  The code for jCanvas 5.1 is at https://github.com/caleb531/jcanvas/blob/1fb5012d8e34bbee12fe3c0bceb3306c47dae938/jcanvas.js#L376
2.  In looking at this older code, I believe the function call in ```drawRect()``` that utilized the 'angle' parameter was ```positionShape()```, which is defined on line 134.

**Comment by caleb531** [comment]
Since jCanvas 5.1, I renamed the `angle` property to `rotate`, so try using that. :)

Caleb

**Comment by AndrewMcMahan** [comment]
Thanks for the reply!

It didn't seem to affect any changes using ```rotate``` either.    I am short of time right now, but I'll see if I can duplicate it in the sandbox later and/or show you some screenshots of the results using this project using ```angle``` with 5.1 and ```rotate``` with the current version.

Just to verify, would ```rotate``` use exactly the same value as the previous ```angle``` did?   (For example, in my snippet in the OP above, you feel I should just be able to substitute the word ```rotate``` for ```angle```?)

**Comment by caleb531** [comment]
Correct; the property was only renamed from angle to rotate, but that behavior was not changed.

I don't know what the value of t is, but I assume it's in degrees? The rotate property expects its value to be in degrees, unless you set inDegrees: false, in which case the value will be assumed to be in radians.

Here's the official rotate example from the jCanvas docs, for what it's worth:
http://projects.calebevans.me/jcanvas/docs/rotateCanvas/#rotating-an-individual-shape

**Comment by AndrewMcMahan** [comment]
Yea, that was the issue -- the value being passed was in radians, not degrees.    I think I now see the cause, it appears as though in 5.1, this author's ```$.jCanvas({...``` initialization in ```$(document).ready``` (which I changed to ```$.jCanvasObject({...```) had a parameter setting of ```inDegrees: false``` which must have, at the time, set a default to all of the other function calls marking them all as "in radians".

Anyway, once I changed ```angle``` to ```radians``` and then added the ```inDegrees: false``` it's all working as before.   Thanks!

**Comment by AndrewMcMahan** [comment]
Oh, just in case you're idly curious, here were the only changes required to update from jCanvas 5.1 to 16.04.01:  https://github.com/AndrewMcMahan/SeatingChartGenerator/commit/7e1d8bd42d7ddd8681cad8a9d3a84757544eb299     ...not too bad for 5 years of changes!

**Comment by caleb531** [comment]
Interesting. I'm surprised `$.jCanvasObject({...})` even worked for you. I deprecated `$.jCanvas` entirely at one point, and `jCanvasObject` is really only meant to be used by me internally (within the source).

The better way to override defaults is like so.

```js
$.extend($.jCanvas.defaults, {
  strokeStyle: '#000',
  strokeWidth: 2,
  x: centerX, y: centerY,
  inDegrees: false
});
```

I realize the docs aren't clear on this; I'll update them now. :)

Caleb

**Comment by AndrewMcMahan** [comment]
You know, I'm sure that it's just ignoring what I have completely right now.   Frankly, I just noticed that ```$.jCanvas({...})``` was causing an error in the browser, so I changed it to what I thought had replaced it just looking at your javascript ..haha.

I'll change it to what you suggested, just since that would be better and more accurate.  However, to be honest, having the parameters explicitly stated for each function call is not a bad idea either, so I'll probably leave most of them.
